homeassistant/components/tuya/number.py

```diff
@@ -260,6 +260,29 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # EV charger
+    "zndb": (
+        NumberEntityDescription(
+            key=DPCode.SETA,
+            name="Charging current",
+            native_unit_of_measurement=UnitOfElectricCurrent.AMPERE,
+            entity_category=EntityCategory.CONFIG,
+        ),
+        NumberEntityDescription(
+            key=DPCode.SETT,
+            name="Charging time",
```

Comment:
How does this entity behave? Does it count down during charging?

Reply:
No, it just count the time during charging.  It's reset chan charge is finished.

Reply:
> No, it just count the time during charging

That isn't a clear answer. Will it go down during charging or not?